pkg/local/diff.go

```diff
@@ -265,7 +265,7 @@ func WalkS3(root string, callbackFunc func(p string, info fs.FileInfo, err error
 // DiffLocalWithHead Checks changes between a local directory and the head it is pointing to. The diff check assumes the remote
 // is an immutable set so any changes found resulted from changes in the local directory
 // left is an object channel which contains results from a remote source. rightPath is the local directory to diff with
-func DiffLocalWithHead(left <-chan apigen.ObjectStats, rightPath string, includeDirs bool) (Changes, error) {
+func DiffLocalWithHead(left <-chan apigen.ObjectStats, rightPath string, includeUnixPermissions bool) (Changes, error) {
```

Comment:
The previous change made sense, however I think we need to leave the parameter name the same.
The includeDirs functionality is something that can be seen as separate from the permissions feature.
For example, even after we deprecate this feature if we decide we want to support directory markers in lakectl local this will still be relevant